ledger/internal/prefetcher/prefetcher_test.go

```diff
@@ -61,7 +63,10 @@ func (l *prefetcherTestLedger) BlockHdr(basics.Round) (bookkeeping.BlockHeader,
 func (l *prefetcherTestLedger) CheckDup(config.ConsensusParams, basics.Round, basics.Round, basics.Round, transactions.Txid, ledgercore.Txlease) error {
 	return nil
 }
-func (l *prefetcherTestLedger) LookupWithoutRewards(_ basics.Round, addr basics.Address) (ledgercore.AccountData, basics.Round, error) {
+func (l *prefetcherTestLedger) LookupWithoutRewards(rnd basics.Round, addr basics.Address) (ledgercore.AccountData, basics.Round, error) {
+	if _, has := l.errorAddress[addr]; has {
+		return ledgercore.AccountData{}, l.round, fmt.Errorf("Lookup Error")
```

Comment:
you might want to use a custom error type here so that you could make `errors.Is` when testing.